src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -396,8 +395,9 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       case Failure(e) =>
         logger.error(s"Preemption work failed for call ${callKey.tag}! " + e.getMessage, e)
         currentSender ! CallActor.Ack(message)
-        self ! CheckForWorkflowComplete
+        self ! CallFailedNonRetryable(callKey, events, returnCode, e.getMessage)
       case Success(_) =>
+        executionStore += keyClone -> ExecutionStatus.NotStarted
```

Comment:
Oh, I think this is actually the wrong order of things, the call should be persisted to Preempted before it creates its clone, because the way it is now if Cromwell is stopped in between we're going to end up with the Preempted call still `Running` and its clone `NotStarted`, which is not good.